vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -0,0 +1,97 @@
+
+
+To create and add tabs to your page, do the following steps:
+
+1. Configure your `components`, see our guide on [Containers](containers.md).
+2. Assign the previously configured `components` to the `tabs` argument of the [`Tabs`][vizro.models.Tabs] model. This creates the connection between your page and the organized tab structure.
+
+!!! example "Simple tabs"
+    === "app.py"
+        ```py
+        import vizro.models as vm
+        import vizro.plotly.express as px
+        from vizro import Vizro
+
+        iris = px.data.iris()
+
+        page = vm.Page(
+            title="Page Title",
+            components=[
+                vm.Tabs(
+                    tabs=[
+                        vm.Container(
+                            title="Tab 1",
+                            components=[
+                                vm.Graph(
+                                    figure=px.scatter(
+                                        iris, x="sepal_length", y="petal_width", color="species"
+                                    )
+                                ),
+                            ]
+                        ),
+                        vm.Container(
+                            title="Tab 2",
+                            components=[
+                                vm.Graph(
+                                    figure=px.bar(
+                                        iris, x="sepal_length", y="sepal_width", color="species"
+                                    )
+                                ),
+                            ]
+                        ),
+                    ]
+                )
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+        - components:
+            -tabs:
+              - components:
+                  - figure:
+                      _target_: scatter
+                      data_frame: iris
+                      x: sepal_length
+                      y: petal_width
+                      color: species
+                    type: graph
+                type: container
+                title: Tab 1
+              - components:
+                  - figure:
+                      _target_: scatter
+                      data_frame: iris
+                      x: sepal_length
+                      y: petal_width
+                      color: species
+                    type: graph
+                type: container
+                title: Tab 2
+              type: tabs
+          title: Page Title
+        ```
+    === "Result"
+        [![Page]][Page]
+
+    [Page]: ../../assets/user_guides/pages/page_sunburst.png
+
+!!! note
+
+    `title` of the `Container` models will appear as title of individual tab components
```

Comment:
```suggestion
    The `title` argument of the `Container` models will appear as the title of the individual tab components
```

Reply:
But maybe we can slot this note into bullet point three above. I think it dies not necessarily need its own note at the bottom of the page